pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java

```diff
@@ -112,13 +110,7 @@ public CmdSinks(Supplier<PulsarAdmin> admin) {
     abstract class BaseCommand extends CliCommand {
         @Override
         void run() throws Exception {
-            try {
-                processArguments();
-            } catch (Exception e) {
-                String chosenCommand = jcommander.getParsedCommand();
-                getUsageFormatter().usage(chosenCommand);
-                throw e;
-            }
+            processArguments();
```

Comment:
Does here changed the origin logic? Or picocli handled the case by default?

Reply:
Let's hand it over to picocli for processing.